chia/wallet/vc_wallet/vc_wallet.py

```diff
@@ -0,0 +1,460 @@
+from __future__ import annotations
+
+import dataclasses
+import logging
+import time
+from typing import TYPE_CHECKING, Any, List, Optional, Set, Tuple, Type, TypeVar, Union
+
+from blspy import G1Element, G2Element
+
+from chia.protocols.wallet_protocol import CoinState
+from chia.server.ws_connection import WSChiaConnection
+from chia.types.announcement import Announcement
+from chia.types.blockchain_format.coin import Coin, coin_as_list
+from chia.types.blockchain_format.program import Program
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.coin_spend import CoinSpend
+from chia.types.spend_bundle import SpendBundle
+from chia.util.ints import uint32, uint64, uint128
+from chia.wallet.did_wallet.did_wallet import DIDWallet
+from chia.wallet.puzzles.p2_delegated_puzzle_or_hidden_puzzle import solution_for_conditions
+from chia.wallet.sign_coin_spends import sign_coin_spends
+from chia.wallet.transaction_record import TransactionRecord
+from chia.wallet.util.compute_memos import compute_memos
+from chia.wallet.util.transaction_type import TransactionType
+from chia.wallet.util.wallet_sync_utils import fetch_coin_spend_for_coin_state
+from chia.wallet.util.wallet_types import AmountWithPuzzlehash, WalletType
+from chia.wallet.vc_wallet.vc_drivers import VerifiedCredential
+from chia.wallet.vc_wallet.vc_store import VCRecord, VCStore
+from chia.wallet.wallet import Wallet
+from chia.wallet.wallet_coin_record import WalletCoinRecord
+from chia.wallet.wallet_info import WalletInfo
+
+_T_VCWallet = TypeVar("_T_VCWallet", bound="VCWallet")
+
+
+class VCWallet:
+    # WalletStateManager is only imported for type hinting thus leaving pylint
+    # unable to process this
+    wallet_state_manager: Any  # pylint: disable=used-before-assignment
```

Comment:
The comment doesn't seem to be correct and the `Any` can probably be replaced after #14900 if its not possible already?

Reply:
fixed